### PR TITLE
Use channel icons for channel cards

### LINKED ISSUE #2168: channels use 'topic' rather than 'channel' icon
image

We use this material icon for channels:

image

### PR DESCRIPTION
**Pull request by christianmemije**
Fixes #2168 

127 0 0 1-8000-learn-

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
       :title="content.title"
       :thumbnail="content.thumbnail"
       :class="{'grid-item': true, 'mobile': isMobile}"
-      :kind="content.kind"
+      :kind="content.id === content.channel_id ? channelKind : content.kind"
```
Comment (jonboiser) [inline kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: Why isn't `content.kind` equal to CHANNEL to begin with for channels?

Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: Are you referring to why it's kind isn't CHANNEL in the back-end?

Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: I can also move this logic to the _topicState(), but that doesn't address the fact that it comes with kind set to topic.

Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: @jonboiser Moved logic to _topicState

Comment (jonboiser) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: Okay, that sounds like a good idea to remove that logic from the component.

So channels have kind 'topic' and the only way to check if a 'topic' is a channel is to check the channel and node id are the same?

Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: Looks like it:

image

Comment (indirectlylit) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: I believe that channels have a "root topic node", and that's the thing we actually interact with in the API.

Comment (indirectlylit) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: It's always seemed weird to me, too. Maybe there's a possibility to refactor.

File: kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue [hunk 2]
```diff
@@ -91,6 +91,9 @@
       channelFilterSelection: '',
     }),
     computed: {
+      channelKind() {
```
Comment (jonboiser) [inline kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: I don't think this method is necessary anymore.

Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/views/content-card-group-grid/index.vue]: Fixed.

**Pull request merged**